content/concept-transaction-cost.md

```diff
@@ -17,14 +17,52 @@ You can also [query `rippled` for the current transaction cost](#querying-the-tr
 The transaction cost is not paid to any party: the XRP is irrevocably destroyed. Since no new XRP can ever be created, this makes XRP more scarce, and consequently benefits all holders of XRP by making XRP more valuable.
 
 
-## Load Scaling ##
+## Load Cost and Open Ledger Cost ##
+
+When the [FeeEscalation amendment](concept-amendments.html#feeescalation) is enabled, there are two thresholds for the transaction cost:
+
+* If the transaction cost does not meet a `rippled` server's [load-based transaction cost threshold](#local-load-cost), the server ignores the transaction completely. (This logic is essentially unchanged with or without the amendment.)
+* If the transaction cost does not meet a `rippled` server's [open ledger cost threshold](#open-ledger-cost), the queues the transaction for the next ledger.
```

Comment:
`the server queues the transaction for a later ledger`

"server" is just a missing word. I suggest "a later" instead of "the next" because there's no telling when a queued transaction will be reattempted. It certainly isn't guaranteed that it will be the next ledger, especially if there are a lot of items in the queue, and a particular transaction is cheap.